Dev/src/components/TopBanner.js

```diff
@@ -28,10 +29,22 @@ class TopBanner extends FontLoadingComponent {
                             onPress={LeftOnPress ? () => { LeftOnPress() } : () => { }}>{LeftTextName}</Text>
                     </TouchableOpacity>
                 </View>
-
                 <View style={styles.topMiddleView}>
+                
+
+                    {(isEditingTitle ?
+                   
+                    <TextInput
+                    value={Title}
+                    onChangeText={onTitleChanged}
+                    //colors.lightGrey
+                    style={{minWidth: screenWidth * 0.40, backgroundColor: "#e5c8b3", padding: 13, borderRadius: 400}}
```

Comment:
For the background color, use the colors we've defined by "import colors from 'config/colors" and using the colors from there.